frontend/src/text_resources.js

```diff
@@ -532,8 +533,9 @@ let LOCALIZE = new LocalizedStrings({
     mainTabs: {
       homeTabTitle: "Accueil",
       dashboardTabTitle: "Tableau de bord",
-      prototypeTabTitle: "Prototype",
-      statusTabTitle: "Statut"
+      sampleTest: "eMIB Sample",
```

Comment:
Maybe we should put _FR_ in front to make sure that we don't forget the translation on that one too?